3.9.1/index.d.ts

```diff
@@ -57,8 +23,23 @@ declare interface Gulp {
    * </ul>
    */
   task: typeof Orchestrator.prototype.add;
-  run: RunMethod;
-  watch: typeof watch;
+  /**
+   * Run the specified task(s)
+   * @deprecated will be removed in gulp 4.0. Use task dependencies instead.
+   */
+  run: typeof Orchestrator.prototype.start;
+  /**
+   * Run the specified task(s)
+   * @deprecated will be removed in gulp 4.0. Use task dependencies instead.
+   */
+  start: typeof Orchestrator.prototype.start;
+  /**
+   * watches all files in the specified glob(s) for changes and runs the given task(s).
+   * @param globs
+   * @param tasks
+   */
+  watch(globs?: string | Array<string>, tasks?: task | Array<task>): EventEmitter;
+  watch(globs?: string | Array<string>, opt?: WatchOptions, tasks?: task | Array<task>): EventEmitter;
```

Comment:
@Haringat It's a bit clearer to use `string[]` over `Array<string>` (if only for conciseness). For more complex type shapes, I'd definitely recommend using the generic syntax. FWIW, the TypeScript type hints will always shorten it to `string[]` regardless I've found, so perhaps that's a reason?